example/nextjs/pages/index.js

```diff
@@ -39,7 +39,7 @@ const Index = ({profile, origin}) => (
       <h2 className='subtitle is-4'>
         Stateless authentication microservice for
         <ul>
-          {['Twitter', 'Facebook', 'Google', 'GitHub', 'Reddit', 'LinkedIn', 'Instagram'].map(name => <Item name={name} key={name} />)}
+          {['Twitter', 'Facebook', 'Google', 'GitHub', 'Reddit', 'LinkedIn', 'Instagram', 'Eventbrite'].map(name => <Item name={name} key={name} />)}
```

Comment:
The login-with.com page (and the related demo code) is not meant to be deployed by anyone else. 

The respective user / developer can configure which strategies to use via environment variables.

To test a specific strategy, you can directly connect (from browser) to your auth endpoint (e.g. auth.login-with.com/twitter or auth.your-service.com/eventbrite)